website/public/docs/quickstart.zh.md

```
@@ -36,6 +41,34 @@ irm https://copaw.agentscope.io/install.ps1 | iex
+>      - 检查 `uv` 是否可用：在 CMD 中输入 `uv --version` ，如果显示版本号，则**只需配置 CoPaw 路径**；如果提示“'uv' 不是内部或外部命令，也不是可运行的程序或批处理文件。”，则需同时配置两者。
+>      - uv路径（任选其一，取决于安装位置，若第一步不可用则填）：通常在`%USERPROFILE%\.local\uv\bin`、`%USERPROFILE%\AppData\Local\uv\bin`或 Python 安装目录下的 `Scripts` 文件夹
+>      - CoPaw路径：通常在 `%USERPROFILE%\.copaw\bin` 。
+>    - **手动添加到系统的 Path 环境变量**：
+> 
+>      - 按 `Win + R`，输入 `sysdm.cpl` 并回车，打开“系统属性”。
+>      - 点击 “高级” -> “环境变量”。
+>      - 在 “系统变量” 中找到并选中 `Path`，点击 “编辑”。
+>      - 点击 “新建”，依次填入上述两个目录路径，点击确定保存。
+> 2. **如果你是用的是 PowerShell（.ps1）：脚本运行中断**
+> 
+>   由于 **受限语言模式** 模式，脚本可能无法自动下载`uv`。
```

markdownlint-cli2 (0.21.0)

[warning] Multiple spaces after blockquote symbol

(MD027, no-multiple-space-blockquote)